src/wp-includes/html-api/class-wp-html-stack-event.php

```diff
@@ -56,14 +56,23 @@ class WP_HTML_Stack_Event {
 	 */
 	public $operation;
 
+	/**
+	 * Indicates if the stack element is a real or virtual node.
+	 *
+	 * @var string
+	 */
+	public $provenance;
+
 	/**
 	 * Constructor function.
 	 *
```

Comment:
It is worth noting that the 3rd parameter was added in WP 6.6.0. Although it's probably enough to put it for the constructor as it seems to be a new class in this release.

Reply:
Yes, given this class was added in 6.6.0, I don't think we need to note the change.